app/scripts/components/datasets/s-overview/index.tsx

```diff
@@ -35,7 +35,7 @@ function DatasetsOverview() {
   return (
     <>
       <LayoutProps
-        title={`${dataset.data.name} Overview`}
+        title={dataset.data.name}
```

Comment:
oh do we not even want it on the document title?

Reply:
Good point - I actually don't know where the document title comes up. We can keep the suffix there, I guess, I'd just separate it from the dataset title, if that's okay.